app/Http/Controllers/Api/ExperienceSkillsController.php

```diff
@@ -19,6 +19,17 @@ public function store(StoreExperienceSkill $request)
             ['experience_id', $data['experience_id']],
             ['experience_type', $data['experience_type']]
         ])->first();
+
+        // Attach skill to applicant if not already attached.
+        $applicant = $request->user()->applicant;
```

Comment:
This is a bit of an edge case, but technically this might be the wrong applicant. An admin could use the api to create an ExperienceSkill attached to a different user's Experience.
We'll need to get the Experience object we're attaching to (see similar code in `Requests/StoreExperienceSkill.php`), then find the experience's Applicant. Note that depending on whether `$experience->experienceable_type` is 'applicant' or 'application', this is a little different: `$experience->experienceable` or `$experience->experienceable->applicant`.

There's already code in `Requests/StoreExperienceSkill.php` for getting an Experience from id and type. We want to use the same code here (and probably elsewhere in the future) so we should extract it a method somewhere. My first thought on where to put it is to create a common base class that all the Experience_____ models extend, and store some common utility methods there. (Actually, you could add another method there to extract getting Applicant from Experience, as well.)